sklearn/partial_dependence.py

```diff
@@ -252,7 +252,7 @@ def partial_dependence(est, target_variables, grid=None, X=None,
     if method not in method_to_function:
         raise ValueError(
             'method {} is invalid. Accepted method names are {}, auto.'.format(
-                method, ', '.join(method_to_function.keys())))
+                method, ', '.join(sorted(method_to_function.keys()))))
```

Comment:
is using the dict worth it if we're only having two functions?

Reply:
like
```python
if method == "recursion":
   # do stuff
   func = _partial_depenence_recursion
elif method == "brute":
   # do stuff
   func = _partial_dependence_brute
else:
   # error
func(...)
```

Reply:
not really, I can remove it